analytics.go

```diff
@@ -143,7 +142,7 @@ func New(key string) *Client {
 		now:      time.Now,
 		uid:      uid,
 	}
-
+	go c.loop()
```

Comment:
I think this was mostly interesting when doing development, if the in-flight message limit is high then the tests likely won't post the events until the program is stopped (which usually happens with a hard kill in dev so the events aren't sent at all). 
I agree that setting the limit low seems like a simpler way of solving this problem.